doc/source/setup-helm.rst

```diff
@@ -65,6 +71,8 @@ cluster:
    This command only needs to run once per Kubernetes cluster, it will create a
    `tiller` deployment in the kube-system namespace and setup your local `helm`
    client.
+   This is the step that configures `helm` commands executed in a local
+   terminal to be deployed within the remote cluster by `tiller`.
```

Comment:
Perhaps this sentence could be rephrased? I'm not a native english speaker but to me the word _deployed_ doesn't fit well enough.
>   This is the step that configures `helm` commands executed in a local
>   terminal to be deployed within the remote cluster by `tiller`.

Perhaps this?
>   This is the step that couples your `helm` commands executed in a local
>   terminal to commands executed by `tiller` in the remote Kubernetes cluster.

Reply:
My mental model is that `helm` the command-line executable on my laptop talks to `tiller` and tells it what to do. Tiller then does it. Assuming that model is correct my suggestion would be:

"This command installs and configures the ``tiller`` part of Helm (the whole project, not the CLI) on the remote kubernetes cluster. Later when you want to deploy changes with ``helm`` (the local CLI) it will talk to tiller and tell it what to do. Tiller then executes these instructions from within the cluster."

Reply:
Thanks @consideRatio , don't worry it did sound a bit clunky to me too so feedback is appreciated!

@betatim I like this explanation, it echos the one at the top of the page and I think this is an important concept to reiterate.